src/Sensor/MemoryMeasurement.java

```diff
@@ -7,18 +7,22 @@
 public class MemoryMeasurement extends Measurement {
     
     MemoryMeasurement(Sigar _sigar){
-        super("Memory measure");
-        sigar = _sigar;
+        super("Memory measure", _sigar);
     }
 
     public String getActualMeasure(){
-        Long value = (long) -1;
+        Long memoryUsageValueBytes = (long) -1;
+        Long memoryUsageValueMegabytes = (long) -1;
+        final Long bytesToKilobytesDivider = 1024L;
+        final Long kilobytesToMegabytesDivider = 1024L;
 
         try {
-            value = sigar.getMem().getActualUsed(); //getActualUsed() returns memory in bytes
-        } catch (SigarException var2) {
-            var2.printStackTrace();
+            memoryUsageValueBytes = sigar.getMem().getActualUsed(); //getActualUsed() returns memory in bytes
+            memoryUsageValueMegabytes = memoryUsageValueBytes / bytesToKilobytesDivider / kilobytesToMegabytesDivider;
```

Comment:
This convertion violates _single responsibilty principe_, consider extracting it and helper constants to separate private (if used only here) static method.